### PR TITLE
upgrade Oracle Instant Client to 19.18

### PR DESCRIPTION
**Pull request by fairclothjm**
Preparing for the upcoming v0.8.0 plugin release

### CODE REVIEW

**Comment by fairclothjm** [comment]
Ooops, going to bump this all the way up to 19.18 which is the latest and greatest

File: Dockerfile [hunk 1]
```diff
@@ -18,7 +18,7 @@ RUN yum update -y && yum install -y  \
 		zip \
 		git
 
-ENV GOLANG_VERSION 1.17.7
+ENV GOLANG_VERSION 1.19.1
```
Comment (swenson) [inline Dockerfile:21]:
```suggestion
ENV GOLANG_VERSION 1.19.5
```

**Review by swenson — DISMISSED** [review]
LGTM

**Review by maxcoulombe — APPROVED** [review]

**Comment by fairclothjm** [comment]
Thanks!

**Pull request merged**